python/ipywidgets/ipywidgets/widgets/widget_output.py

```diff
@@ -134,15 +132,6 @@ def __exit__(self, etype, evalue, tb):
             if ip:
                 kernel = ip
                 ip.showtraceback((etype, evalue, tb), tb_offset=0)
-            elif self.comm is not None and self.comm.kernel is not None:
-                kernel = self.comm.kernel
-                kernel.send_response(kernel.iopub_socket,
-                                     u'error',
-                                     {
-                    u'traceback': ["".join(traceback.format_exception(etype, evalue, tb))],
-                    u'evalue': repr(evalue.args),
-                    u'ename': etype.__name__
-                    })
```

Comment:
Why can we remove this?

Reply:
I will bring it back. What is annoying with this code is that it's very specific to ipykernel, but I can add more checks so we don't go through this with *e.g.* xeus-python